auacm/app/database.py

```diff
@@ -1,18 +1,17 @@
-'''Database handlers.'''
+'''Creates the database.'''
 from sqlalchemy import create_engine
 from sqlalchemy.orm import Session
 from sqlalchemy.ext.automap import automap_base
 
 # Create global database variables
-Base = automap_base()
-engine = create_engine('mysql+pymysql://acm@localhost/acm?charset=utf8')
-connection = engine.connect()
-session = Session(engine)
+DATABASE_BASE = automap_base()
```

Comment:
I'm not a fan of keeping it as-is. Uppercasing constants should be reserved for immutable objects that are assigned once.

I'd prefer 3, but I recognize that that's a lot of work for a single refactoring. For now, I'd go with 1 and leave a comment to disable the inspection.